core/storage/collection/gae_models.py

```diff
@@ -226,7 +205,7 @@ def get_export_policy():
 
        TODO(#8523): Implement this function.
        """
-        return base_models.EXPORT_POLICY.TO_BE_IMPLEMENTED
+        return base_models.EXPORT_POLICY.NOT_APPLICABLE
```

Comment:
Why isn't this whole method replaced with `pass` as in the other files?

Reply:
Fixed